simplecpp.cpp

```diff
@@ -2426,8 +2426,16 @@ std::map<std::string, simplecpp::TokenList*> simplecpp::load(const simplecpp::To
             continue;
 
         std::ifstream fin(filename.c_str());
-        if (!fin.is_open())
+        if (!fin.is_open()) {
+            if (outputList) {
+                simplecpp::Output err(fileNumbers);
+                err.type = simplecpp::Output::ERROR;
+                err.location = Location(fileNumbers);
+                err.msg = "Can not open include file '" + filename + "' that is explicitly included for all files.";
```

Comment:
Unless you can motivate "for all files" please remove that. Imagine this TU has 1 source file and 4 headers that are included. I do not believe that the file is included 5 times in this TU.

Reply:
You are right, I meant only source files so the message is not correct. Maybe it is best to remove "for all files".